### ISSUE TITLE
'TrafficManager' object has no attribute 'global_distance_to_leading_vehicle'

### ISSUE THREAD

**Issue by ll7** [carla-simulator/carla#3841]
## Error

```
Exception has occurred: AttributeError
'TrafficManager' object has no attribute 'global_distance_to_leading_vehicle'
  File "/home/luttkule/git/carla_optimization/simple_scenario.py", line 15, in <module>
    tm.global_distance_to_leading_vehicle(1)
```
## code to replicate
```python
import carla
# import time

client = carla.Client('localhost', 2000)
client.set_timeout(10.0) # seconds
# client.reload_world()

world = client.get_world()
world = client.load_world('Town03')

# traffic manager
tm = client.get_trafficmanager(8000)
tm.set_random_device_seed(0)
tm.global_percentage_speed_difference(-5.0)
tm.global_distance_to_leading_vehicle(1)
...
```

## Software Version
os: ubuntu 20.04
python: 3.8.5
carla: 0.9.10.1 binary download

## references

[traffic manger doc](https://carla.readthedocs.io/en/0.9.10/adv_traffic_manager/#creating-a-traffic-manager)

[traffic manager python api](https://carla.readthedocs.io/en/0.9.10/python_api/#carlatrafficmanager)

**Comment by ll7** [comment]
https://github.com/carla-simulator/carla/blob/0c97e9a5de5b35b759cc5b0955801244ed76791f/PythonAPI/examples/spawn_npc.py#L112

uses a set method, which is not listed in the python api reference and not in the example.

**Comment by jackbart94** [comment]
It's fixed now, but you are checking the 0.9.10v of documentation. We will make sure that one is correct as well.
